ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -499,6 +499,30 @@ static int HandleVnodeOperation(
                         kauthError))
                 {
                     goto CleanupAndReturn;
+                }                
+            }
+            
+            if(ActionBitIsSet(action, KAUTH_VNODE_WRITE_DATA))
+            {
+                if (!TryGetVirtualizationRoot(&perfTracer, context, currentVnode, pid, &root, &vnodeFsidInode, &kauthResult, kauthError))
+                {
+                    goto CleanupAndReturn;
+                }
+                
+                PerfSample fileModifiedSample(&perfTracer, PrjFSPerfCounter_VnodeOp_FileModified);
+                
+                if (!TrySendRequestAndWaitForResponse(
+                                                      root,
+                                                      MessageType_KtoU_NotifyFileModified,
+                                                      currentVnode,
+                                                      vnodeFsidInode,
+                                                      vnodePath,
+                                                      pid,
+                                                      procname,
+                                                      &kauthResult,
+                                                      kauthError))
```

Comment:
I'm very much nit-picking here, but this isn't the whitespace convention for multi-line function calls we've been following in the code so far. Rather than aligning arguments with the opening parenthesis, we've simply been indenting them by one or more regular indentation steps. As part of an `if` condition, we've been indenting the arguments by an extra 1-2 levels, depending on how it looks I guess. If it's not in an `if`, one indentation level will usually do.

I'm assuming Xcode added the whitespace for you. If I remember correctly, the setting to turn off for this is Preferences… -> Text Editing -> Indentation -> "Automatically indent based on syntax". I don't think you can turn it off on a per-project or per-file basis. (The setting works reasonably well with Objective-C code, but generates unreasonable amounts of whitespace in C and C++ code.)

Reply:
Good catch and tip!  Yes I should definitely be sticking to convention.